experimental/Schemes/CoherentSheaves.jl

```diff
@@ -146,6 +148,7 @@ identifications given by the glueings in the `default_covering`.
   ID::IdDict{AbsSpec, ModuleFP} # the modules on the basic patches of the default covering
   OOX::StructureSheafOfRings # the structure sheaf on X
   M::PreSheafOnScheme # the underlying presheaf of modules for caching
+  C::Covering # The covering of X on which the modules had first been described
```

Comment:
you call this 'default covering' a few lines before, maybe you should stay consistent with this to avoid confusion.

Reply:
But that would overwrite a function's name with a local variable. So: No, I don't want to do that.

Reply:
I am refering to the help text (after the  #) not to the name of it